client/reader/stream/index.jsx

```diff
@@ -379,7 +388,7 @@ class ReaderStream extends Component {
 
 			props.trackScrollPage( pageId );
 		}
-		const pageHandle = stream.pageHandle || { before: startDate };
+		const pageHandle = this.getPageHandle( stream.pageHandle, startDate );
```

Comment:
Just curiosity what prompted change here? For the most part it seems to be doing the same thing as before except now will return `null` instead of `{ before: null|undefined|etc. }` in those cases where both pageHandle and startDate are falsy. Seems better, i was just curious what prompted the change in this file (caught an existing bug, other changes in this PR caused issue due to this?)

Reply:
the `{before: undefined }` was causing the API to throw a 400 error on the new `/cards` endpoint